br/pkg/lightning/importer/import.go

```diff
@@ -357,7 +362,26 @@ func NewImportControllerWithPauser(
 		regionSizeGetter := &local.TableRegionSizeGetterImpl{
 			DB: db,
 		}
-		backendConfig := local.NewBackendConfig(cfg, maxOpenFiles, p.KeyspaceName)
+
+		versionStr, _ := version.FetchVersion(ctx, db)
+		if err := version.CheckTiDBVersion(versionStr, minTiDBCurrentResourceGroupVersion, maxTiDBCurrentResourceGroupVersion); err != nil {
+			if !berrors.Is(err, berrors.ErrVersionMismatch) {
+				return nil, common.ErrCheckKVVersion.Wrap(err).GenWithStackByArgs()
+			}
+			log.FromContext(ctx).Debug("TiDB version doesn't support get current resource group.", zap.Error(err))
+		} else {
+			// get resource group name.
+			exec := common.SQLWithRetry{
+				DB:     db,
+				Logger: log.FromContext(ctx),
+			}
+
+			if err := exec.QueryRow(ctx, "", "select current_resource_group();", &p.ResourceGroupName); err != nil {
```

Comment:
I prefer execute this query without check the version and ignore certain errors.

Reply:
done.